### PR TITLE
feat: [#1295] fix subscription issue

### LINKED ISSUE #1295: Fix subscription issue - live update

### PR DESCRIPTION
**Pull request by wiktorflorian**
Closes #1295

### CODE REVIEW

File: transform/app/services/jms/connector.py [hunk 1]
```diff
@@ -166,8 +198,10 @@ async def connect(
         conn.disconnect()
     except ConnectFailedException as e:
         logger.error(f"Failed to connect to the JMS: {host}:{port}, error: {e}")
+        raise
```
Comment (Michal-Kolomanski) [inline transform/app/services/jms/connector.py:201]: good catch!

File: transform/app/services/jms/connector.py [hunk 2]
```diff
@@ -64,30 +67,54 @@ def __init__(
...
-            self._ssl,
-        )
+        asyncio.run_coroutine_threadsafe(self.handle_reconnection(), self.event_loop)
+
+    async def handle_reconnection(self):
+        """
+        Asynchronously handle reconnection attempts. Waits for a set interval before trying to reconnect.
+        Continues attempts until a successful reconnection is established.
+        """
+        await asyncio.sleep(INIT_JMS_RETRY_INTERVAL)
```
Comment (Michal-Kolomanski) [inline transform/app/services/jms/connector.py:100]: we shouldn't wait here. Reconnect ASAP

**Review by Michal-Kolomanski — APPROVED** [review]

File: transform/app/services/jms/connector.py (hunk 2 above)
Comment (wiktorflorian) [reply transform/app/services/jms/connector.py]: Done

File: transform/app/services/jms/connector.py (hunk 1 above)
Comment (wiktorflorian) [reply transform/app/services/jms/connector.py:200]: Thank you <3

**Pull request merged**